src/odin/narrativebuilder.cc

```diff
@@ -464,6 +464,31 @@ void NarrativeBuilder::Build(std::list<Maneuver>& maneuvers) {
         maneuver.set_verbal_arrive_instruction(FormVerbalArriveInstruction(maneuver));
         break;
       }
+      case DirectionsLeg_Maneuver_Type_kElevatorEnter: {
```

Comment:
What about the verbal instructions?

Reply:
Thanks for reviewing this PR @dgearhart ! Yeah, I wasn't sure what's the best verbal instructions for these indoor maneuvers, and I left it blank with the intention that this could be iterated later on. 
But I could take a stab at it too. I propose for `verbal_pre_transition_instruction`, we can stick to the same instructions as `instruction` while for `verbal_post_transition_instruction`, we can use `FormVerbalPostTransitionInstruction()`.  What do you think? Let me know what your thoughts!